common/src/main/scala/com/gu/media/model/PlutoIntegrationData.scala

```diff
@@ -79,6 +79,38 @@ object PlutoSyncMetadataMessage {
   }
 }
 
+case class PlutoResyncMetadataMessage(
+   `type`: String,
+    enabled: Boolean,
```

Comment:
Don't think we need this property.

Reply:
I have kept the properties that are not being used to avoid the need to do any special processing on the other side of the transaction. pluto may be expecting these (it definitely is in the case of userEmail, it uses it to map the user into a user ID for tracking) so if they are not present that may break things downstream.
I am keen not to add any more complexity in processing than necessary, since this is simply a replay of an existing action I don't really want to be having alternate code paths in the handling of the message

Reply:
`enabled` is a legacy feature and should/will be removed as its now controlled by configuration.

> it definitely is in the case of userEmail, it uses it to map the user into a user ID for tracking

The issue here though is we're setting the email to `system@video.gutools.co.uk` which does not relate to an Editorial user...

Reply:
hmmm, that is true. in that case Pluto just logs the email address.  More importantly, I don't want it falling over because it is expecting to have that key in the sent message

Reply:
tbh, I have checked through the pluto code that handles this and it looks like nothing reads that property.  From what you're saying it should be deprecated on _all_ the messages, not just this one, right?  Would you prefer that in a seperate PR or on the end of this one?

Reply:
related PR https://github.com/guardian/media-atom-maker/pull/749